tests/src/core/parse.js

```diff
@@ -21,4 +22,37 @@ describe('parse(content, { settings, ecmaFeatures })', function () {
       .not.to.throw(Error)
   })
 
+  it('passes expected parserOptions to custom parser', function () {
+    const parseSpy = sinon.spy()
+    const parserOptions = { ecmaFeatures: { jsx: true } }
+    require('./parseStubParser').parse = parseSpy
```

Comment:
Right, can pull to top. Both this and resolve. Just a habit of having self-contained tests.